src/app/ngrx/reducers/group-charted.reducer.ts

```diff
@@ -27,22 +27,6 @@ export function reducer(
       }, state);
     }
 
-    // when ranks are loaded, add them to state
```

Comment:
I changed this reducer to only store the groups that have been toggled off/on by the user and selectors that use this store now assume that groups are charted implicitly unless set to false here in this store. This is related to the `isGroupCharted` function you'll see elsewhere.